presto-druid/src/test/java/io/prestosql/plugin/druid/TestDruidIntegrationSmokeTest.java

```diff
@@ -272,4 +287,73 @@ public void testFilteringForTablesAndColumns()
         actualColumns = computeActual("DESCRIBE " + datasourceB);
         Assert.assertEquals(actualColumns, expectedColumns);
     }
+
+    @Test
+    public void testAggregationPushdown()
+    {
+        assertAggregationPushedDown("SELECT count(*) FROM orders");
+
+        // for varchar only count is pushed down
+        assertAggregationPushedDown("SELECT count(comment) FROM orders");
+
+        // for timestamp
+        assertAggregationPushedDown("SELECT count(__time) FROM orders");
+        assertAggregationPushedDown("SELECT min(__time) FROM orders");
+        assertAggregationPushedDown("SELECT max(__time) FROM orders");
+
+        // for double
```

Comment:
I think grouping tests by agg function name makes more sense.
Then, if you add support for a new functon, you just add a new section here.

Reply:
i see i didn't follow the rule myself in postgres tests.... up to you